src/MarketingPageAcceptanceTests/MarketingPageAcceptanceTests.Objects/Utils/RandomGenerator.cs

```diff
@@ -0,0 +1,39 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+using OpenQA.Selenium.Internal;
+
+namespace MarketingPageAcceptanceTests.Objects.Utils
+{
+    public class RandomGenerator
+    {
+        private Random random = new Random();
+        private const string Characters = @"abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ1234567890!£$%^&*(){}:@~<>?|[];#,./";
```

Comment:
change variable name to be a little more descriptive (maybe `AllCharacters` or similar)